guvnor-ala/guvnor-ala-wildfly/guvnor-ala-wildfly-provider/src/main/java/org/guvnor/ala/wildfly/executor/WildflyRuntimeExecExecutor.java

```diff
@@ -28,61 +28,70 @@
 import static com.google.common.base.Strings.isNullOrEmpty;
 import static org.guvnor.ala.util.RuntimeConfigHelper.buildRuntimeName;
 
-public class WildflyRuntimeExecExecutor<T extends WildflyRuntimeConfiguration> implements RuntimeBuilder<T, WildflyRuntime>,
-        RuntimeDestroyer,
-        FunctionConfigExecutor<T, WildflyRuntime> {
+public class WildflyRuntimeExecExecutor<T extends WildflyRuntimeConfiguration>
+        implements RuntimeBuilder<T, WildflyRuntime>,
+                   RuntimeDestroyer,
+                   FunctionConfigExecutor<T, WildflyRuntime> {
 
     private final RuntimeRegistry runtimeRegistry;
     private final WildflyAccessInterface wildfly;
-    protected static final Logger LOG = LoggerFactory.getLogger( WildflyRuntimeExecExecutor.class );
+    protected static final Logger LOG = LoggerFactory.getLogger(WildflyRuntimeExecExecutor.class);
 
     @Inject
-    public WildflyRuntimeExecExecutor( final RuntimeRegistry runtimeRegistry,
-            final WildflyAccessInterface wildfly ) {
+    public WildflyRuntimeExecExecutor(final RuntimeRegistry runtimeRegistry,
+                                      final WildflyAccessInterface wildfly) {
         this.runtimeRegistry = runtimeRegistry;
         this.wildfly = wildfly;
     }
 
     @Override
-    public Optional<WildflyRuntime> apply( final WildflyRuntimeConfiguration config ) {
-        final Optional<WildflyRuntime> runtime = create( config );
-        if ( runtime.isPresent() ) {
-            runtimeRegistry.registerRuntime( runtime.get() );
+    public Optional<WildflyRuntime> apply(final WildflyRuntimeConfiguration config) {
+        final Optional<WildflyRuntime> runtime = create(config);
+        if (runtime.isPresent()) {
+            runtimeRegistry.registerRuntime(runtime.get());
         }
         return runtime;
     }
 
-    private Optional<WildflyRuntime> create( final WildflyRuntimeConfiguration runtimeConfig ) throws ProvisioningException {
+    private Optional<WildflyRuntime> create(final WildflyRuntimeConfiguration runtimeConfig) throws ProvisioningException {
 
         String warPath = runtimeConfig.getWarPath();
-        final Optional<WildflyProvider> _wildflyProvider = runtimeRegistry.getProvider( runtimeConfig.getProviderId(), WildflyProvider.class );
+        final Optional<WildflyProvider> _wildflyProvider = runtimeRegistry.getProvider(runtimeConfig.getProviderId(),
+                                                                                       WildflyProvider.class);
 
         WildflyProvider wildflyProvider = _wildflyProvider.get();
```

Comment:
see that there are a considerable amount of code that I in some way I've "inherited" plus millions of lines of code for implementing the UI, so yes, it's not possible to do a complete refactoring/covering of the inherited code at the same time I develop the UI, plus all the "missing" pieces for implementing the UI itself.
I'll look at this line, but, I believe we should focus on the LA. See that the LA will even not include the WF10 stuff.